src/VisualStudio/IntegrationTest/TestUtilities/Interop/AutomationRetry/RawElementProviderSimple.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using UIAutomationClient;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.Interop.AutomationRetry
+{
+    internal class RawElementProviderSimple : AutomationRetryWrapper<IRawElementProviderSimple>, IRawElementProviderSimple
```

Comment:
Did you write an analyzer to generate these from the managed interfaces, or did you do it all by hand?

Reply:
:regex: